src/app/components/addOffer/moreAboutUs/MoreAboutUs.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Submit from '../../additions/SubmitButton';
+import Main from './main/Main'
+import Footer from './footer/Footer'
+
+const MoreAboutUs = ({ pl }) => {
+    return (
+        <div className="moreAboutUs">
+            <Main pl={pl} />
+            <Submit
+                divClass={'buttonCenter'}
```

Comment:
```suggestion
                divClass="buttonCenter"
```
to zadziała? przyznam, że nie pamiętam